database/migrations/2020_12_08_180857_create_experiences_table.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateExperiencesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('experiences', function (Blueprint $table) {
+            $table->id();
+            $table->string('title');
+            $table->dateTime('start_date');
+            $table->dateTime('end_date');
+            $table->string('company');
+            $table->longText('description');
+
+            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('industry_category_id');
+            $table->unsignedBigInteger('career_level_id');
+
+            $table->timestamps();
+            $table->foreign('user_id')->references('id')
+                ->on('users')->onDelete('cascade')
+                ->onUpdate('cascade');
+
+            $table->foreign('industry_category_id')->references('id')
+                ->on('industry_categories')->onDelete('cascade')
```

Comment:
`onDelete("set null")` in stead of `cascade` and also needless to say make the column nullable